resources/functionMap.php

```diff
@@ -11251,24 +11251,24 @@
 'SplEnum::getConstList' => ['array', 'include_default='=>'bool'],
 'SplFileInfo::__construct' => ['void', 'file_name'=>'string'],
 'SplFileInfo::__toString' => ['string'],
-'SplFileInfo::getATime' => ['int'],
+'SplFileInfo::getATime' => ['int|false'],
```

Comment:
Can you wrap these in `__benevolent<...>`? Like this? `__benevolent<int|false>`

Because when SplFileInfo comes from a class like Symfony Finder, we can be sure those file exist, so having to check against `false` would be too annoying.

Reply:
i tired.